addons/mail/static/src/messaging/component/activity/activity.xml

```diff
@@ -9,14 +9,17 @@
                         <t t-if="activity.assignee">
                             <img class="o_Activity_userAvatar" t-attf-src="/web/image/res.users/{{ activity.assignee.id }}/image_128" t-att-alt="activity.assignee.nameOrDisplayName"/>
                         </t>
-                        <i class="o_Activity_icon fa"
+                        <div class="o_Activity_icon"
                             t-att-class="{
                                 'bg-success-full': activity.state === 'planned',
                                 'bg-warning-full': activity.state === 'today',
                                 'bg-danger-full': activity.state === 'overdue',
                             }"
-                            t-attf-class="{{ activity.icon }}"
-                        />
+                        >
+                            <i class="fa"
+                                t-attf-class="{{ activity.icon }}"
```

Comment:
icon is a field in DB so I don't think it's a good idea to edit that field... Maybe a getter on the entity or something else ?

Reply:
This is an issue. I guess it must be upgraded in DB with the fontawesome update, otherwise it is not going to work. Please check in PR 48094 if he converted it, and if so, how he did it.

Reply:
As I see, he updated all activity icons with the far/fas + icon name so activity.icon should contain all needed classes when the PR 48094 is merge (and the class="fa" could be removed).

Do we wait the PR 48094 to be merged (and our branch rebased) to make that change or do we do it "prevently" ?

Reply:
As discussed, change data in DB as done in the linked PR and preemptively remove `fa` from our code.